dracut/repartition/endless-repartition.sh

```diff
@@ -80,9 +80,18 @@ if [ -z "${root_disk}" ]; then
   exit 0
 fi
 
+pt_label=$(blkid -o value -s PTTYPE $root_disk)
+
 # Check for our magic "this is Endless" marker
-marker=$(sfdisk --force --part-attrs $root_disk $partno)
-if [ "$marker" != "GUID:55" ]; then
+if [ "$pt_label" = "dos" ]; then
+  marker=$(sfdisk --force --part-type $root_disk 4)
```

Comment:
Right. Working on this after a while made me forget these details. 
We are _still_ using the partition `4` for this, so this line is actually correct. Thank for reminding this.